### DISCUSSION TITLE
Can't see deforum in my tabs

### DISCUSSION THREAD

**Discussion by davidtimwaldman2** [deforum-art/deforum-stable-diffusion#366] [General]
Has anyone else had this issue? I've installed and updated my extensions. I've reinstalled SD webui. Totally stuck!

**Comment by HappyRobotLtd** [comment]
Hi,

Same here - very annoying - this seems to be a V1.8.1 issue since it was working only last week.

No amount of reinstalling SD / Deforum / ControlNet makes any difference.

**Reply by HappyRobotLtd** [reply]
*I'm using SD Automatic 1111 via Google Colab - is there a similar issue with local installs ?
